app/src/main/java/org/jellyfin/mobile/MainActivity.kt

```diff
@@ -125,18 +128,26 @@ class MainActivity : AppCompatActivity(), WebViewController {
                 val url = request.url
                 val path = url.path?.toLowerCase(Locale.ROOT) ?: return null
                 return when {
-                    path.endsWith(Constants.INDEX_PATH) -> {
-                        val patchedIndex = loadPatchedIndex(httpClient, url.toString())
-                        if (patchedIndex != null) {
-                            runOnUiThread { connectionHelper.onConnectedToWebapp() }
-                            patchedIndex
-                        } else {
-                            runOnUiThread { connectionHelper.onErrorReceived() }
-                            emptyResponse
+                    path.endsWith("scripts/apploader.js") -> {
+                        runOnUiThread {
+                            webView.evaluateJavascript(
+                                """
+                                !function() {
+                                    var scripts = ['native/nativeshell.js', 'native/apphost.js', 'native/EventEmitter.js', 'native/chrome.cast.js'];
+                                    scripts.forEach(function(src) {
+                                        var scriptElement = document.createElement("script");
+                                        scriptElement.src = src;
+                                        scriptElement.setAttribute("async", "");
+                                        document.head.appendChild(scriptElement);
```

Comment:
```suggestion
                                        document.body.appendChild(scriptElement);
```

Injections are in body too